roles/matrix-bot-maubot/defaults/main.yml

```diff
@@ -17,11 +17,9 @@ matrix_bot_maubot_bot_server_public: "https://{{ matrix_server_fqn_matrix }}"
 matrix_bot_maubot_proxy_management_interface: false
 matrix_bot_maubot_expose_management_interface: true
 
-
-matrix_bot_maubot_secret: ''
-matrix_bot_maubot_admin_user: ''
-matrix_bot_maubot_admin_password: ''
-matrix_mau_environment_variables_extension: ''
+matrix_bot_database_uri: 'sqlite:///data/maubot.db'
+matrix_bot_maubot_port: 29316
+matrix_bot_maubot_secret: 'generate'
```

Comment:
I think the default for port and database are fine. I don't see why you added `matrix_bot_maubot_secret` as this should be set by a user and it would be unsafe to have a default value. We currently enforce the user to set it manually which is fine I think